src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+// import React, { useState } from 'react';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="display">0</div>
+    <div className="row">
+      <button type="button" className="btn clear">AC</button>
+      <button type="button" className="btn plus-minus">+/-</button>
+      <button type="button" className="btn percent">%</button>
+      <button type="button" className="btn operator">&divide;</button>
```

Comment:
- [ ] [OPTIONAL] Since one of the strengths of React is reusable components, you can make your code DRY by creating a `<Button />` component which you can reuse in your app 👍